sandbox/sandbox_json.cpp

```diff
@@ -320,13 +320,13 @@ enum Bla
 template <class Archive>
 void save( Archive & ar, Bla const & b )
 {
-  ar( (const int &)b );
+  ar( static_cast<const int &>(b) );
```

Comment:
I think a lot of the code in the sandbox files was temporary for debugging development, especially early on in cereal. Most tests of note are put into actual test cases now.